src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -38,6 +41,8 @@
 
 "ExposureDetection_RefreshingIn" = "Update in %02d:%02d minutes";
```

Comment:
maybe add non-breakable space here, same goes for the German file

Reply:
done